tool/site/client/src/pages/loadPage.tsx

```diff
@@ -40,34 +46,45 @@ const LoadPage: React.FC = () => {
   const [prescriptionFilesUploaded, setPrescriptionFilesUploaded] = useState([])
   const [prescriptionsInTestPack, setPrescriptionsInTestPack] = useState([])
   const [loadFormValues, setLoadFormValues] = useState<LoadFormValues>()
-  const [loadPageErrors, setLoadPageErrors] = useState<LoadPageErrors>()
+  const [loadPageErrors, setLoadPageErrors] = useState<LoadPageErrors>({details:[]})
 
   useEffect(() => {
     (async() => {
       if (loadFormValues) {
 
-        setLoadPageErrors(undefined)
+        setLoadPageErrors({details: []})
 
         const bundles = await getBundles(baseUrl, loadFormValues, prescriptionsInTestPack, prescriptionFilesUploaded)
 
         if (!bundles.length) {
-          setLoadPageErrors({details: "Unable to read prescription(s)"})
+          setLoadPageErrors({details: ["Unable to read prescription(s)"]})
         }
 
         bundles.forEach(bundle => {
           updateBundleIds(bundle)
           updateValidityPeriod(bundle)
         })
 
-        const response = await (await axiosInstance.post<LoadResponse>(`${baseUrl}prescribe/edit`, bundles))
-        const responseData = getResponseDataIfValid(response, isLoadResponse)
-        window.location.href = encodeURI(responseData.redirectUri)
+        const loadResponses = await uploadBundlesInBatchesOfTen(bundles)
+
+        window.location.href = encodeURI(loadResponses[0].redirectUri)
+      }
+
+      async function uploadBundlesInBatchesOfTen(bundles: Bundle[]) {
+        const loadResponses: Array<LoadResponse> = []
+        const chunkSize = 10
+        for (let i = 0; i < bundles.length; i += chunkSize) {
+          const chunk = bundles.slice(i, i + chunkSize)
+          const response = await axiosInstance.post<LoadResponse>(`${baseUrl}prescribe/edit`, chunk)
+          loadResponses.push(getResponseDataIfValid(response, isLoadResponse))
+        }
+        return loadResponses
```

Comment:
super minor but could we use consistent naming here for batch/chunk e.g. its called `uploadBundlesInBatchesOfTen` so line 75 is `const batchSize = 10` and line 77 is `const batch = bundles.slice(i, i + batchSize)` or vice versa